### PR TITLE
openCypher Query and Visualization Support

### PR DESCRIPTION
**Pull request by michaelnchin**
# openCypher Query and Visualization Support
With the release of support for the openCypher query language in Amazon Neptune's lab mode, graph-notebook can now be used to execute and visualize openCypher queries with any compatible graph database.

Two new magic commands have been added: 
- `%%oc`/`%%opencypher` 
- `%%oc_status`/`%%opencypher_status`

These openCypher magic commands inherit the majority of the query and visualization customization features that are already available in the Gremlin and SPARQL magics.

For more detailed information and examples of how you can execute and visualize openCypher queries through graph-notebook, please refer to the `Air-Routes-openCypher` and `EPL-openCypher` sample notebooks that have been added in this update.

Screen Shot 2021-07-27 at 3 34 29 PM

## Additional Updates
- Overhauled all Gremlin visualization sample notebooks with example usage of new visualization customization options and the `elementMap` step
- Updated Neptune ML notebooks, utils, and pretrained models config
- Added support for `modeltransform` commands in `%neptune_ml`
- Refactored %load form display code for flexibility; fixes some descriptions being cut off

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -81,6 +81,7 @@ def get_version():
         'botocore>=1.19.37',
         'boto3>=1.17.58',
         'ipython>=7.16.1',
+        'neo4j',
```
Comment (bechbd) [inline setup.py:84]: Do we/should we pin the version here?

File: src/graph_notebook/seed/queries/propertygraph/identity/00_identity_group_nodes_gremlin.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...").property("type","household").addV("identityGroup").property(id,"003bea55c68c7625555c43c3375b141fd154210f").property("igid","003bea55c68c7625555c43c3375b141fd154210f").property("type","household").addV("identityGroup").property(id,"003d7adb821efac06d6de7aef472175233d0c1fa").property("igid","003d7adb821efac06d6de7aef472175233d0c1fa").property("type","household").addV("identityGroup").property(id,"003d906d79e3534d7c3ca7ff58a0205186255d3b").property("igid","003d906d79e3534d7c3ca7ff58a0205186255d3b").property("type","household").addV("identityGroup").property(id,"004090ec584eb447086e5370a442da6e9538820e").property("igid","004090ec584eb447086e5370a442da6e9538820e").property("type","household").addV("identityGroup").property(id,"0044ce3eb555ccb356eaf83040eec2f17fb435e7").property("igid","0044ce3eb555ccb356eaf83040eec2f17fb435e7").property("type","household").addV("identityGroup").property(id,"0047bdcec10d7329fda1efc3af6f798b0fa1291f").property("igid","0047bdcec10d7329fda1efc3af6f798b0fa1291f").property("type","household").addV("identityGroup").property(id,"0048499bcd5b698d92a9d0f95669f05048227c5e").property("igid","0048499bcd5b698d92a9d0f95669f05048227c5e").property("type","household")
```
Comment (bechbd) [inline src/graph_notebook/seed/queries/propertygraph/identity/00_identity_group_nodes_gremlin.txt:1]: I don't think these files are supposed to be included in the package due to size

File: src/graph_notebook/seed/queries/propertygraph/identity/01_persistent_nodes_gremlin.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
...operty(id,"380c44b896026385dfd13655eb527380995694d1").property("pid","380c44b896026385dfd13655eb527380995694d1").addV("persistentId").property(id,"6d9fcf1e367fe13bd7405ddb17483f425a8d6944").property("pid","6d9fcf1e367fe13bd7405ddb17483f425a8d6944").addV("persistentId").property(id,"f1a42fd42be8f88695e1a6fced77b10e01ab43d0").property("pid","f1a42fd42be8f88695e1a6fced77b10e01ab43d0").addV("persistentId").property(id,"5c3000a90eb38eeb34da252fd675aea17f915f5b").property("pid","5c3000a90eb38eeb34da252fd675aea17f915f5b").addV("persistentId").property(id,"8ecf79339d8df1e00dea903be00f6013821a70e6").property("pid","8ecf79339d8df1e00dea903be00f6013821a70e6").addV("persistentId").property(id,"7a976baa095047b5fcaa97e939fd723058b4c06a").property("pid","7a976baa095047b5fcaa97e939fd723058b4c06a").addV("persistentId").property(id,"c092d73a469471b9303e9d3c5b755f362d2106be").property("pid","c092d73a469471b9303e9d3c5b755f362d2106be").addV("persistentId").property(id,"73a5add082977b7d1588f51ab5e057907e6402a7").property("pid","73a5add082977b7d1588f51ab5e057907e6402a7").addV("persistentId").property(id,"6b12fd2aa1638468206b47c2d9d53e3cabd1afff").property("pid","6b12fd2aa1638468206b47c2d9d53e3cabd1afff")
```
Comment (bechbd) [inline src/graph_notebook/seed/queries/propertygraph/identity/01_persistent_nodes_gremlin.txt:1]: I don't think these files are supposed to be included in the package due to size

**Review by bechbd — CHANGES_REQUESTED** [review]

File: src/graph_notebook/seed/queries/propertygraph/identity/00_identity_group_nodes_gremlin.txt (hunk 2 above)
Comment (michaelnchin) [reply src/graph_notebook/seed/queries/propertygraph/identity/00_identity_group_nodes_gremlin.txt]: Removed

File: src/graph_notebook/seed/queries/propertygraph/identity/01_persistent_nodes_gremlin.txt (hunk 3 above)
Comment (michaelnchin) [reply src/graph_notebook/seed/queries/propertygraph/identity/01_persistent_nodes_gremlin.txt]: Removed

File: setup.py (hunk 1 above)
Comment (michaelnchin) [reply setup.py]: Yes; pinned to latest version, since we know that is currently working

**Review by bechbd — APPROVED** [review]

**Review by krlawrence — APPROVED** [review]

**Pull request merged**